src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OverloadMethodsDeclarationOrderCheck.java

```diff
@@ -29,16 +29,29 @@
 
 /**
  * <p>
- * Checks that overload methods are grouped together.
+ * Checks that overload methods are grouped together. Overloaded methods have same
```

Comment:
minor, grammatical suggestion: "Checks that overload**ed** methods are grouped together. Overloaded methods have **the** same"

Reply:
thanks, I will update

Reply:
This is updated.